valkey.conf

```diff
@@ -1011,7 +1002,7 @@ replica-priority 100
 # * stream - Data type: streams related.
 #
 # For more information about ACL configuration please refer to
-# the Redis web site at https://redis.io/topics/acl
+# the server web site at https://redis.io/topics/acl
```

Comment:
```suggestion
# the server web site at https://valkey.io/topics/acl
```

though this might still not work atm.

Reply:
There are multiple `redis.io` references and I think we should leave it as is for the moment.

Reply:
Sure. We can do a follow up PR to clean these up but I think we need these to be fixed in RC1

Reply:
Unsure if we have secured a domain or else I can submit a PR linking to our github repo at least for the user to land on an appropriate page.

@madolson

Reply:
"the server web site" -> "the web site" or "the Valkey website"

We have the domain name.